crates/ruff_cli/src/lib.rs

```diff
@@ -69,6 +69,10 @@ quoting the executed command, along with the relevant file contents and `pyproje
         }));
     }
 
+    // Enabled ANSI colors on Windows 10.
+    #[cfg(windows)]
+    let _ = colored::control::set_virtual_terminal(true);
```

Comment:
According to the docs this always returns `Ok`, and the signature is just a backwards-compatibility thing. Should we `.unwrap()`?

Reply:
I think yes, if it returns something other than `Ok` that is an error which would be silenced otherwise.

Reply:
I would assume that the LTO removes the error case.

I would prefer an assert over unwrap since we're not interested in the value

assert!(...is_ok())